src/main/index.ts

```diff
@@ -235,6 +235,40 @@ app.whenReady().then(async () => {
       MAIN_WINDOW.on('close', mainWindowCloseHandler);
     }
   });
+  ipcMain.handle('open-logs', async () => KANGAROO_FILESYSTEM.openLogs());
+  ipcMain.handle('export-logs', async () => KANGAROO_FILESYSTEM.exportLogs());
+  ipcMain.handle('factory-reset', async () => {
+    const userDecision = await dialog.showMessageBox({
+      title: 'Factory Reset',
+      type: 'warning',
+      buttons: ['Cancel', 'Confirm'],
+      defaultId: 0,
+      cancelId: 0,
+      message: `Are you sure you want to fully reset ${KANGAROO_CONFIG.productName}? This will delete all your ${KANGAROO_CONFIG.productName} related data.`,
```

Comment:
```suggestion
      message: `Are you sure you want to factory reset ${KANGAROO_CONFIG.productName}? This will delete all  ${KANGAROO_CONFIG.productName} data in all profiles. This cannot be undone.`,
```

Just a suggestion -- feel free to ignore if you prefer your wording.

Reply:
I think I would mention the profile name explicitly. This is an advanced feature anyway and its good to be super clear about what you're deleting.

Reply:
I'm just a bit vary of the ambiguity of the term "profile" here...It may be interpreted that it's got something to do with a profile one may have set up in the app itself (in the sense of avatar/nickname).

Reply:
What do you think about that? https://github.com/holochain/kangaroo-electron/commit/0636439d30015b1906b749ae3edc881f58ae01b3